common/frame.go

```diff
@@ -1033,12 +1027,8 @@ func (f *Frame) getAttribute(selector, name string, opts *FrameBaseOptions) (goj
 	if err != nil {
 		return nil, err
 	}
-	gv, ok := v.(goja.Value)
-	if !ok {
-		return nil, fmt.Errorf("unexpected type %T", v)
-	}
 
-	return gv, nil
+	return asGojaValue(f.ctx, v), nil
```

Comment:
Same here, maybe it's ok to panic instead of erroring, since i believe all errors are eventually turned into a panic

Reply:
Yep, I noticed it and fixing it now, thanks.